TwitchClient/GamesListViewController.swift

```diff
@@ -1,108 +0,0 @@
-//
-//  GamesListViewController.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/7/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-import SVProgressHUD
-
-class GamesListViewController: UIViewController, WebserviceManagerDelegate, UITableViewDelegate, UITableViewDataSource {
```

Comment:
The protocols can be implemented as extensions or better yet as separate objects, to keep the VC clean.